unime/src/i18n/nl-NL/index.ts

```diff
@@ -170,20 +140,57 @@ const nl = {
     LOG_OUT: {
       TITLE: 'Log uit',
     },
+    THEME: {
+      SYSTEM: 'Systeem',
+      LIGHT: 'Licht',
+      DARK: 'Donker',
+    },
+    PASSWORD: {
+      POLICY: {
+        TITLE: 'Wachtwoord',
+        UPPERCASE_LETTER: 'Hoofdletters',
+        LOWERCASE_LETTER: 'Kleine letters',
+        NUMBER: 'Nummers',
+        CHARACTERS: 'Karakters',
+      },
+    },
+    RESET_APP: {
+      TITLE: 'App opnieuw instellen',
+      DESCRIPTION: 'Weet je zeker dat je de app opnieuw wilt instellen en alle gegevens wilt verwijderen?',
+      CONFIRM: 'Ja, verwijder alles',
+      CANCEL: 'Nee, bewaar mijn profiel',
+    },
+    ACCOUNT: 'Profiel',
+    SUPPORT: {
+      TITLE: 'Support',
+      ABOUT: {
+        TITLE: 'Over UniMe',
+        NAVBAR_TITLE: 'Over UniMe',
+        BUILT_WITH: 'Ontwikkeld met Tauri',
+      },
+      FEEDBACK: {
+        TITLE: 'Stuur feedback',
+      },
+    },
   },
   LOCK_SCREEN: {
     PASSWORD_INPUT_PLACEHOLDER: 'Voer je wachtwoord in',
     BUTTON_TEXT: 'Ontgrendel wallet',
     FORGOT_PASSWORD: 'Wachtwoord vergeten?',
   },
   ME: {
+    BOTTOM_NAVIGATION_TITLE: 'Me',
     GREETINGS: {
       GREETING_0: 'Hey',
       GREETING_1: 'Hallo',
       GREETING_2: 'Welkom terug',
       GREETING_3: 'Hoi',
       GREETING_4: 'Dag',
     },
+    DEMO: {
+      TEXT_1: 'Bezoek',
+      TEXT_2: 'op een desktopcomputer om te beginnen.',
+    },
```

Comment:
Looks like these labels are not used in the code yet, So that means that the English hardcoded translations are still being used.

You'll find the hardcoded English text in `unime/src/lib/credentials/CredentialList.svelte` and in `unime/src/routes/(app)/me`

Reply:
Should be fixed :)